### DISCUSSION TITLE
How to use NavbarNested

### DISCUSSION THREAD

**Discussion by sandeepsre** [mantinedev/mantine#2062] [Q&A]
I am trying to using NavbarNested but linking is not working. please guide me how to pass links into nested navbar

**Comment by rtivital** [comment]
Hi @sandeepsre, Mantine UI provides examples of prebuilt components, it provides full source code and mockdata for these components on the website and GitHub. Unfortunately, we cannot provide support for Mantine UI as it usually takes a lot of time. You can browse Mantine UI source code (https://github.com/mantinedev/ui.mantine.dev) to find everything related to components.

**Reply by sandeepsre** [reply]
thanks

**Reply by sandeepsre** [reply]
any other git repo related to NavbarNested?
